source/Calamari.Testing/EnvironmentVariables.cs

```diff
@@ -70,31 +77,46 @@ public enum ExternalVariable
         [EnvironmentVariable("CALAMARI_AUTHPASSWORD", "OctopusMyGetTester")]
         MyGetFeedPassword,
 
-        [EnvironmentVariable("GOOGLECLOUD_OCTOPUSAPITESTER_JSONKEY", "GoogleCloud - OctopusAPITester")]
+        [EnvironmentVariable("GOOGLECLOUD_OCTOPUSAPITESTER_JSONKEY", "GoogleCloud - OctopusAPITester", "op://Calamari Secrets for Tests/Google Cloud Octopus Api Tester JsonKey")]
         GoogleCloudJsonKeyfile,
-        
+
         [EnvironmentVariable("GitHub_RateLimitingPersonalAccessToken", "GitHub test account PAT")]
         GitHubRateLimitingPersonalAccessToken,
     }
 
     public static class ExternalVariables
     {
+        static readonly bool SecretManagerIsEnabled = Convert.ToBoolean(Environment.GetEnvironmentVariable("CALAMARI__Tests__SecretManagerEnabled") ?? "True");
```

Comment:
You _may_ want to stick with `OCTOPUS__Tests__SecretManagerEnabled`. _Maybe_ a benefit in consistency?